### PR TITLE
contributing to the AI section - issue: Add some new resources #145 

### PR DESCRIPTION
**Pull request by elidakirigo**
# 🛠️ Fixes issue #145 
# 📇 Changes proposed 
# ✅ Check List (Mark all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] This PR contains changes in the documentation
- [x] This PR contains changes in the Code of the project
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by elidakirigo** [comment]
Add some new resources #145

File: pages/AI-resources.md [hunk 1]
```diff
@@ -62,3 +61,13 @@
 |  2. Git and GitHub crash course (freecodecamp) | <center> [![Git and GitHub crash course (freecodecamp)](https://i.ytimg.com/vi/RGOj5yH7evk/mqdefault.jpg "Git and GitHub crash course (freecodecamp)")](https://www.youtube.com/watch?v=RGOj5yH7evk "Git and GitHub crash course (freecodecamp)") <center>  | [Here you go](https://www.youtube.com/watch?v=RGOj5yH7evk)
 |  3. Git for Professionals (freecodecamp) | <center> [![Git for Professionals (freecodecamp)](https://i.ytimg.com/vi/Uszj_k0DGsg/mqdefault.jpg "Git for Professionals (freecodecamp)")](https://www.youtube.com/watch?v=Uszj_k0DGsg "Git for Professionals (freecodecamp)") <center>  | [Here you go](https://www.youtube.com/watch?v=Uszj_k0DGsg)
 ---
+
+## Problem Solving Challenges
+
+|Description | Preview   | Link to get started with it   |
+| :------------: | :------------: | :------------: |
+|  1. Google Hash Code 2020 | <center> [![Complete Git and GitHub Tutorial](https://i.ytimg.com/vi/Mewe008IGl0/mqdefault.jpg "Google Hash Code 2020")](https://www.youtube.com/watch?v=Mewe008IGl0 "Google Hash Code 2020") <center>  | [Here you go](https://www.youtube.com/watch?v=Mewe008IGl0)
```
Comment (hasnainmakada-99) [inline pages/AI-resources.md:69]: @elidakirigo, You've put the resources which are to be inside a playlist, please remove these single resources and if there is a whole playlist of hashCode on google, Please update it with the whole playlist here

**Review by hasnainmakada-99 — CHANGES_REQUESTED** [review]

**Comment by hasnainmakada-99** [comment]
And make sure to please propose the new changes which you made inside the file with reference to that file 👇🏻 and also checklist all the necessary check boxes @elidakirigo 
image

File: pages/AI-resources.md (hunk 1 above)
Comment (elidakirigo) [reply pages/AI-resources.md:69]: okay @hasnainmakada-99

**Comment by elidakirigo** [comment]
a new pull request is submitted for this issue.

regards.

**Pull request closed without merging**